src/HubBootstrapper.php

```diff
@@ -0,0 +1,180 @@
+
+        $clientBuilder = new ClientBuilder($this->options);
+        $clientBuilder
+            ->setTransportFactory($this->transportFactory)
+            ->setLogger($this->logger);
+
+        $client = $clientBuilder->getClient();
+
+        $hub = $this->hub;
+        $hub->bindClient($client);
+        SentrySdk::setCurrentHub($hub);
+    }
+
+    /**
+     * @param IntegrationInterface[] $integrations
+     *
+     * @return IntegrationInterface[]
+     */
+    public function prepareIntegrations(array $integrations)
+    {
+        $userIntegrations = $this->resolveIntegrationsFromUserConfig();
+        if ($this->options->hasDefaultIntegrations()) {
+            $integrations = array_filter(
+                $integrations,
+                static function (
+                    SdkIntegration\IntegrationInterface $integration
+                ): bool {
+                    if (
+                        $integration instanceof
+                        SdkIntegration\ErrorListenerIntegration
+                    ) {
+                        return false;
+                    }
+
+                    if (
+                        $integration instanceof
+                        SdkIntegration\ExceptionListenerIntegration
+                    ) {
+                        return false;
+                    }
+
+                    if (
+                        $integration instanceof
+                        SdkIntegration\FatalErrorListenerIntegration
+                    ) {
+                        return false;
+                    }
+
+                    // We also remove the default request integration so it can be readded
+                    // after with a Yii3 specific request fetcher. This way we can resolve
+                    // the request from Yii3 instead of constructing it from the global state
+                    return ! ($integration instanceof
+                        SdkIntegration\RequestIntegration);
+                }
+            );
+
+            $integrations[] = new SdkIntegration\RequestIntegration(
+                new YiiRequestFetcher($this->container)
+            );
+        }
+
+        return array_merge($integrations, $userIntegrations);
+    }
+
+    /**
+     * Resolve the integrations from the user configuration with the container.
+     *
+     * @return SdkIntegration\IntegrationInterface[]
+     */
+    private function resolveIntegrationsFromUserConfig(): array
+    {
+        // Default Sentry SDK integrations
+        $integrations = [
+            new Integration(),
+            new Integration\ExceptionContextIntegration(),
+        ];
+
+        $integrationsToResolve = $this->configuration->getIntegrations();
+
+        $enableDefaultTracingIntegrations = isset(
+            $this->configuration
+                ->getTracing()['default_integrations']
+        )
+            ? (bool)$this->configuration->getTracing()['default_integrations']
+            : true;
+
+        if (
```

Comment:
Sorry, can't understand the logic behind that.

Reply:
if config has ['default_integrations']=> true, then enabled HubBootstrapper::DEFAULT_INTEGRATIONS